plugin/src/main/java/io/jenkins/plugins/casc/ConfigurationContext.java

```diff
@@ -34,12 +39,21 @@
 
     public ConfigurationContext(ConfiguratorRegistry registry) {
         this.registry = registry;
+        String prop = Util.fixEmptyAndTrim(System.getProperty(
+            CASC_YAML_MAX_ALIASES_PROPERTY,
+            System.getenv(CASC_YAML_MAX_ALIASES_ENV)
+        ));
+        yamlMaxAliasesForCollections = NumberUtils.toInt(prop, 50);
```

Comment:
@timja Please see 57aebdd commit message for my reasoning to use a property.

> This needs to be a property/env since the context is not yet configurable as it has yet to read any YAML
> and it might be blocked due to alias limit.
> 
> Adding the reading of the property/env to ConfigurationContext
> makes sense to avoid YamlUtils to constantly reading the property on every file read
> when it attempts to merge all the sources.